src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String CIRCLE_COLOR = "white";
+    private static final int CIRCLE_RADIUS = 10;
+    private static final int MAX_VALUE = 10;
+    private int[] param = new int[3];
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(FigureTypes.values().length);
+        FigureTypes figureType = FigureTypes.values()[index];
+        String color = new ColorSupplier().getRandomColor();
+
+        for (int i = 0; i < param.length; i++) {
+            param[i] = new Random().nextInt(MAX_VALUE);
+        }
+
+        switch (figureType) {
+            case CIRCLE:
+                return new Circle(color, param[0]);
```

Comment:
```suggestion
                return new Circle(color, random.nextInt(MAX_VALUE));
```
And better create private methods for creating figures (getRandomCircle(), etc)